extensions/configuration-editing/src/extension.ts

```diff
@@ -61,11 +64,25 @@ function registerSettingsCompletions(): vscode.Disposable {
 	});
 }
 
-function newSimpleCompletionItem(text: string, range: vscode.Range, description?: string): vscode.CompletionItem {
+function registerExtensionsCompletions(): vscode.Disposable {
+	return vscode.languages.registerCompletionItemProvider({ pattern: '**/extensions.json' }, {
+		provideCompletionItems(document, position, token) {
+			const location = getLocation(document.getText(), document.offsetAt(position));
+			const range = document.getWordRangeAtPosition(position) || new vscode.Range(position, position);
+			if (location.path[0] === 'recommendations') {
+				return vscode.extensions.all
+					.filter(e => !e.id.startsWith('vscode.'))
```

Comment:
@sandy081 
* Excluded  `Microsoft.vscode-markdown`. Can I exclude all packages starting with `Microsoft.vscode-` ?
* Intelli-sense should not show already entered extensions: OK
* Maybe it's better remove: ExtensionsConfigurationSchema / defaultSnippets / Example ?